File: 2032-largest-odd-number-in-string/2032-largest-odd-number-in-string.java
class Solution {
    public String largestOddNumber(String num) {
        int index = num.length()-1;
        while(index>=0){
            int res = num.charAt(index)-'0';
            if(res%2==1){
                return num.substring(0,index+1);
            }
            index--;
        }
        return "";
    }
}

File: 2032-largest-odd-number-in-string/2032-largest-odd-number-in-string.py
class Solution(object):
    def largestOddNumber(self, num):
        """
        :type num: str
        :rtype: str
        """
        index = len(num)-1
        while(index>=0):
            res = int(num[index])
            if(res%2==1):
                return num[0:index+1]
            index-=1
        return ""
        